aspnetcore/mvc/views/overview.md

```diff
@@ -35,13 +35,13 @@ Use [layouts](xref:mvc/views/layout) to provide consistent webpage sections and
 Views help to establish a [**S**eparation **o**f **C**oncerns (SoC) design](http://deviq.com/separation-of-concerns/) within an MVC app by separating the user interface markup from other parts of the app. Following SoC design makes your app modular, which provides several benefits:
 
 * The app is easier to maintain because it's better organized. Views are generally grouped by app feature. This makes it easier to find related views when working on a feature.
-* The parts of the app aren't tightly coupled. You can build and update the app's views separately from the business logic and data access components. You can modify the views of the app without necessarily having to update other parts of the app.
+* The parts of the app are loosely coupled. You can build and update the app's views separately from the business logic and data access components. You can modify the views of the app without necessarily having to update other parts of the app.
```

Comment:
Hyphenate "loosely coupled"

Reply:
mmmmm ... hummmmm 🤔 ... "Loosely" is an adjective modifying the noun "coupled." There's no possibility of confusion (i.e., it isn't a compound adjective). I think I read something once about overly hyphenating (no pun intended ... *overly-hypenating* :smile:) *-ly* ending word couplets, where the meaning is *completely clear* (I'm having toooo much fun today with this! lol :smile: I've gone into *SillyRex*:tm: mode today). I think I'd leave it ... @Rick-Anderson what do you prefer? I'd consult my *Chicago Manual of Style*, but it's buried in a box for a theoretical move back to SoCal. lol :grinning:

Reply:
http://www.grammarbook.com/punctuation/hyphens.asp
https://en.oxforddictionaries.com/punctuation/hyphen

Reply:
I give the author(s) of Merriam-Webster credit: They spend a *full page* on this in their writer's handbook, and they say that the rules are **all over the darn place**. I think it's really going to come down to "Did MS go with *Chicago*?" I think they did, right? What does *Chicago* say? My copy is really buried atm. ....... However, there is no confusion with the construction of "loosely coupled," so there shouldn't be any dire need for a hyphen. I'd be much more concerned if we were looking at a compound adjective here.